Tests/Api/Report/BaseReportTest.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+
+namespace Tests\Werkspot\BingAdsApiBundle\Tests\Api\Report;
+
+
+use BingAds\Reporting\ReportFormat;
+use BingAds\Reporting\ReportLanguage;
+use BingAds\Reporting\ReportRequest;
+use Werkspot\BingAdsApiBundle\Api\Report\BaseReport;
+
+class BaseReportTest extends \PHPUnit_Framework_TestCase
+{
+    public function testSetFormat()
+    {
+        $report = new BaseReport();
+        $this->assertNull($report->getRequest()->Format);
+
+        $report->setFormat(ReportFormat::Csv);
+        $this->assertEquals(ReportFormat::Csv, $report->getRequest()->Format);
+    }
+
+    public function testSetReturnOnlyCompleteData()
+    {
+        $report = new BaseReport();
+        $this->assertNull($report->getRequest()->ReturnOnlyCompleteData);
+
+        $report->setReturnOnlyCompleteData(true);
```

Comment:
Check the BaseReport class to add docblocks, and make the fluent thing compatible (probably remove it)